.circleci/config.yml

```diff
@@ -110,3 +110,41 @@ workflows:
           filters:
             branches:
               only: /^feature\/.*/
+      - app-center/mapping:
+          name: deploy-test-app-stg-mapping
+          file: mapping/staging/mapping.txt
+          app: $APP_CENTER_APP_NAME
+          token: $APP_CENTER_TOKEN
+          version-name: $APP_CENTER_APP_NAME
+          version-code: 1
```

Comment:
Is this correct? Doesn't this mean that the mappings will all be uploaded to the same version?

Reply:
Yeah, currently we are using `1` as default value for all `version-code` in appcenter.

Reply:
But is that going to cause problems between versions? Can App Center distinguish the mappings based on just the version name?

Reply:
Yeah as the [official document](https://docs.microsoft.com/en-us/appcenter/diagnostics/android-deobfuscation) says to match it with build's Gradle configuration, but we are not changing `versionCode` but the `versionName` https://github.com/rakutentech/android-miniapp/blob/master/testapp/build.gradle#L13 🤔